docs/troubleshooting/Windows/index.md

```diff
@@ -0,0 +1,34 @@
+```{eval-rst}
+.. image:: ../../images/cropped-plus3it-logo-cmyk.png
+   :width: 140px
+   :alt: Powered by Plus3 IT Systems
+   :align: right
+   :target: https://www.plus3it.com
+```
+<br>
+
+# Windows Log-Files
+
+When using `watchmaker` on Windows servers, the primary log-file of interest will be the:
+
+- [`c:\watchmaker\logs\watchmaker.log`](c_watchmaker_logs_watchmaker.log.md)
+- [`c:\watchmaker\logs\salt_call.debug`](c_watchmaker_logs_salt_call.debug.md)
```

Comment:
This was kinda hard to see that the items were links. It was a bit more clear on the Linux side, so cribbing the language and setup from that page.

```suggestion
```{toctree}
:maxdepth: 1
c_watchmaker_logs_watchmaker.log.md
c_watchmaker_logs_salt_call.debug.md
```

Reply:
Yeah. I'm not super-jazzed by how Sphinx renders fixed-width fonts in links. It _should_ just do what standard markdown and standard HTML do (which is to say "use link-text _color_ for links, even if the font-style is fixed-width).